nextstrain/cli/net.py

```diff
@@ -33,6 +33,11 @@ def resolve_host(host: str, family: AddressFamily = AF_UNSPEC) -> Set[Union[IPv4
     specific address family can be chosen by providing *family*.
     """
     return {
-        ip_address(getnameinfo(sockaddr, NI_NUMERICHOST)[0])
+        ip_address(
+            # TODO: Remove ignore if getnameinfo type signature is updated to
+            # handle all possible sockaddr types.
+            # <https://github.com/python/typeshed/issues/13472>
+            getnameinfo(sockaddr, NI_NUMERICHOST)[0] # type: ignore
```

Comment:
> It makes sense for getnameinfo to take as input whatever is output from  
> its inverse function, getaddrinfo, but its type signature has not been  
> updated. Ignore for now.  

Not just type signature, but actual implementation. And I don't expect that to happen because it makes no sense to when the whole point is that IPv6 is disabled.